src/pages/Facility/services/inventory/StockLotSelector.tsx

```
@@ -226,19 +269,37 @@ export default function StockLotSelector({
               return (
                 <div
                   key={inv.id}
-                  className="flex items-center space-x-2 cursor-pointer p-2 hover:bg-accent"
-                  onClick={() => toggleLotSelection(inv.id)}
+                  className={`flex items-center space-x-2 p-2 ${
+                    isDisabled
+                      ? "cursor-not-allowed opacity-50 bg-gray-50"
+                      : "cursor-pointer hover:bg-accent"
+                  }`}
+                  onClick={() => toggleLotSelection(inv.id, isRestricted)}
                 >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)